mass-deployer/pkg/mass-deployer/deployer.go

```diff
@@ -83,7 +83,7 @@ func RunDeployer(ctx context.Context, cfg Config, output string, debug bool) err
 	fmt.Println(string(outputBytes))
 
 	if output == "" {
-		return nil
+		output = "output.yaml"
```

Comment:
![image](https://github.com/threefoldtech/tfgrid-sdk-go/assets/64129/9d376935-965d-4758-b9df-40a48734873c)
you could have used output.yaml as the default value in the output flag for the deploy command